### DISCUSSION TITLE
where can i turn of email verification please can't login anymore :-(

### DISCUSSION THREAD

**Discussion by FreeSpirit63** [ngoduykhanh/PowerDNS-Admin#1612] [General]
How can i turn off the mail verification, on the commend line what file have have to find? 
Since I actived it, I can not login anymore it asks me to verify my email as admin :-(

Robert

**Comment by FreeSpirit63** [comment]
found it in SQL

**Comment by drkmark** [comment]
hi there, it isn't so easy. i'm facing the same issue and cannot simply find it in the db. i believe that this option should be editable somewhere in cfg files, ok strongly hope at least. checking them one by one, if anyone successfuly found it, let me know, pls. i'm gonna be crazy soon and don't wanna reinstall whole pdnsadmin just because of that.

**Reply by FreeSpirit63** [reply]
to be honest since i was like you i looked all over, could be i used Heidisql it loads the db's and they can be edited

https://www.heidisql.com/

Freespirit63

**Comment by FreeSpirit63** [comment]
to be honest since i was like you i looked all over, could be i used Heidisql it loads the db's and they can be edited

Freespirit63

**Reply by FreeSpirit63** [reply]
https://www.heidisql.com/

**Reply by drkmark** [reply]
thanks a lot! interesting tool, but I haven't found any connection to this in the database itself. finally, reinstalling powerdnsadmin has solved it immediately. I really spent a lot of time on that one and today I lost all hope definitely. works like a charm and I'm not angry anymore. ;)
